docs/concepts/benchmarking.md

```diff
@@ -81,9 +81,9 @@ For ner, FL takes only 1.3% longer wall time than centralized-1/3, and only 0.1%
 
 #### CCFRAUD
 <p align="center">
-    <img src="../pics/cc_time.jpg" alt="ccfraud training time" width="600"/>
```

Comment:
don't forget to delete the old files, they don't seem to have been deleted from the PR?